tests/bluetooth/audio/codec/src/main.c

```diff
@@ -1701,7 +1701,7 @@ ZTEST(audio_codec_test_suite, test_bt_audio_codec_cap_meta_set_lang)
 	struct bt_audio_codec_cap codec_cap = BT_AUDIO_CODEC_CAP(
 		BT_HCI_CODING_FORMAT_LC3, 0x0000, 0x0000, {},
 		{BT_AUDIO_CODEC_DATA(BT_AUDIO_METADATA_TYPE_LANG, 'e', 'n', 'g')});
-	char new_expected_data[] = "deu";
+	unsigned char new_expected_data[] = "deu";
```

Comment:
Hmm, this is a bit tricky. Technically `bt_audio_codec_cap_meta_set_lang` takes a `uint8_t` array which is of course unsigned, which I assume was the issue. 

However, we assign a string to it, and `string.h` and most string-related functions use `char` (without unsigned) as the type. 

By making this `unsigned char` it would now conflict with any string functions, which isn't really a problem in this case, but would be in many other usecases. 

If we really want to be type-true here, the type should probably be `const uint8_t []` instead (ditto for the above)

Reply:
whatever makes you and the compiler happy :-) changed to to `const uint8_t[]`